### PR TITLE
Recognize integer multicharacter constants like 'ABCD'

### PR DESCRIPTION
**Pull request by yaroslav-o**

### CODE REVIEW

**Comment by yaroslav-o** [comment]
@eliben

File: tests/test_c_lexer.py [hunk 1]
```diff
@@ -448,11 +448,11 @@ def test_char_constants(self):
         self.assertLexerError("'", ERR_UNMATCHED_QUOTE)
         self.assertLexerError("'b\n", ERR_UNMATCHED_QUOTE)
         self.assertLexerError("'\\xaa\n'", ERR_UNMATCHED_QUOTE)
-
-        self.assertLexerError(r"'\12a'", ERR_INVALID_CCONST)
```
Comment (eliben) [inline tests/test_c_lexer.py]: Not sure why you're modifying these tests, without adding any new tests

Comment (yaroslav-o) [reply tests/test_c_lexer.py]: Added new tests.

**Comment by eliben** [comment]
Thanks for adding the information.

**Pull request merged**